confluent_kafka/src/confluent_kafka.c

```diff
@@ -1529,6 +1529,8 @@ rd_kafka_conf_t *common_conf_setup (rd_kafka_type_t ktype,
          * When both args and kwargs are present the kwargs take
          * precedence in case of duplicate keys.
          * All keys map to configuration properties.
+         *
+         * Copy configuration dict to avoid manipulating application config
```

Comment:
nit (but that's what you want..): How about a trailing period?

Reply:
touché